web/src/main/resources/application-dev-fss.properties

```diff
@@ -8,4 +8,13 @@ oidc.sts.well.known.url=https://security-token-service.nais.preprod.local/.well-
 # OIDC/OPENAM
 oidc.open.am.well.known.url=https://isso-q.adeo.no/isso/oauth2/.well-known/openid-configuration
 oidc.open.am.client.id=fpformidling-q1
-# oidc.open.am.client.secret=<leses inn fra vault ved oppstart>
\ No newline at end of file
+# oidc.open.am.client.secret=<leses inn fra vault ved oppstart>
+
+## Klienter
+# Eksterne (lokal)
+dokdist.rest.base.url=http://dokdistfordeling-q1.teamdokumenthandtering/rest/v1
+journalpost.rest.v1.url=http://dokarkiv-q1.teamdokumenthandtering/rest/journalpostapi/v1/journalpost
+journalpost.rest.proxy.v1.url=http://dokarkivproxy-q1.teamdokumenthandtering/rest/journalpostapi/v1/journalpost
+# Eksterne (felles)
+organisasjon.rs.url=https://modapp-q1.adeo.no/ereg/api/v1/organisasjon
+sak.rs.url=http://sak-q1.teamdokumenthandtering/api/v1/saker
```

Comment:
trenger du sak.rs.url?

Reply:
Ja. Vi henter saksnummer derfra og sjekker om den er lavere enn 15000000. Dette var en slags workaround for noe, men noe usikker om det er fortsatt relevant. Må nok ta en sjekk med deg. 
Men det var den sak med journalføring og saksnummere generert av arkiv og fpsak sener som var vanskelig å skille.

Reply:
Jan Erik slettet sak-klient og den sjekken. Finnes ikke lenger. :-) Søk etter bruk av sak.rs.url

Reply:
Fjernet